pnpm-lock.yaml

```diff
@@ -1,4 +1,4 @@
-lockfileVersion: '6.1'
+lockfileVersion: '6.0'
```

Comment:
I noticed that I'm not a single one who faced that pnpm changes version of lockfile. Maybe we should stop changing it back to 6.1 and let it be 6.0. Is there any different between thsse two?

Reply:
I don't think we have been manually changing it, I think different contributor's environments are automatically changing it whenever they change deps. What version are you on?

```➜  ~ pnpm --version
8.6.1
```

Reply:
![Screenshot_20230828_213607_GitHub.jpg](https://github.com/pingdotgg/uploadthing/assets/55754540/5aab34f5-eab1-430d-ab61-063f8cab22c0)

Theo changed it back to 6.1 not so long ago

Reply:
Personally I had been on almost latest versions of pnpm. Updated to latest couple of days ago